homeassistant/components/modbus/modbus.py

```diff
@@ -169,43 +169,52 @@ async def async_stop_modbus(event: Event) -> None:
 
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, async_stop_modbus)
 
+    UNIT = "unit"
```

Comment:
Don't create a new constant... There is one already there: `ATTR_UNIT`

Reply:
No it is removed from const, because it is no longer to be used generally.

Reply:
> No it is removed from const, because it is no longer to be used generally.

You should not remove it from const. You can just add a comment to indicate that it will be removed.
And you still need it in the tests.